guides/manual-installation.md

```diff
@@ -0,0 +1,279 @@
+## Scope
+
+This documents covers the step to manually install trento server without relying on the trento helm chart or kubernetes all together. The lastest
+available version of SUSE Linux Enterprise Server for SAP Applications is used as the base operating system, which is 15 SP5 at the time of writing.
+
+## List of dependencies
+
+- prometheus
+- postgresql
+- rabbitmq
+- docker runtime
+
+## Installation
+
+### Install prometheus (Optional)
+
+#### Option 1: Use existing installation
+
+If you already have a prometheus server running, you can skip this step. , you can skip the inst
+
```

Comment:
How should the reader install Prometheus ? Maybe adding some commands or a reference to official docs?

Reply:
I added a link to the official prometheus site, in case the user doesn't want to follow our suggestions to install it, he can go there and follow instructions there.